src/cryptonote_core/miner.cpp

```diff
@@ -136,6 +145,80 @@ namespace cryptonote
       return true;
     });
 
+    // if mining and paused ...
+    if( m_is_background_mining_enabled && is_mining() && m_pausers_count > 0 )
```

Comment:
I like using the member variable accessor here ... can't unintentionally modify the variable ...